### ISSUE TITLE
13Gen 13700k, is it safe just upgrade microcode to 0x129 without BIOS upgrade?

### ISSUE THREAD

**Issue by chopin1998** [intel/Intel-Linux-Processor-Microcode-Data-Files#84]
i have upgrade to 0x129, need i upgrade bios together?

**Comment by teoberi** [comment]
How did you update?
Early loading microcode does not work in this case!
https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/issues/78#issuecomment-2286857023

**Comment by chopin1998** [comment]
> How did you update? Early loading microcode does not work in this case! [#78 (comment)](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/issues/78#issuecomment-2286857023) PS https://wccftech.com/intel-0x12b-microcode-patch-fix-14th-13th-gen-cpu-instability-issues/

![image](https://github.com/user-attachments/assets/c2dce69f-0abe-463f-bea1-b1232232d31c)

before upgrade intel-microcode package, this value is 0x123

after upgraded(and reboot) it's now 0x129.

**Comment by teoberi** [comment]
This method does not work!
The microcode must be loaded from the BIOS!

**Comment by teoberi** [comment]
Dell has already released a new BIOS version for to correct the instability problem. (0x129 -> 0x12B)

**Comment by whpenner** [comment]
Yes, just to confirm, in this case, the microcode update must be loaded from BIOS to be effective for the instability issue.

Some issues can be resolved when the microcode update is loaded, but other issues, such as this one, require resetting hardware after loading the microcode update.  Unfortunately, that is only supported in BIOS.
